### PR TITLE
Update arcam_fmj.markdown

### PR DESCRIPTION
**Pull request by kernelpanic85**
## Proposed change
 Add support note for JBL and AudioControl processors

## Type of change
- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/arcam_fmj.markdown [hunk 1]
```diff
@@ -41,6 +41,12 @@ Supported devices:
 - SA 30
 - SR 250
 
+This integration may also work with JBL and AudioControl receivers and processors as they share the same firmware as Arcam. Please add models as support is confirmed:
```
Comment (frenck) [inline source/_integrations/arcam_fmj.markdown]:
```suggestion
This integration may also work with JBL and AudioControl receivers and processors as they share the same firmware as Arcam.
```

File: source/_integrations/arcam_fmj.markdown [hunk 2]
```diff
@@ -41,6 +41,12 @@ Supported devices:
...
+
+JBL:
+
```
Comment (frenck) [inline source/_integrations/arcam_fmj.markdown]: This is already implied by the sentence above.

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @kernelpanic85 👍 

../Frenck

**Comment by elupus** [comment]
@kernelpanic85 do they report them self as some of the other series? We may need to map their reported name to its supported features in the library.

**Comment by kernelpanic85** [comment]
> @kernelpanic85 do they report them self as some of the other series? We may need to map their reported name to its supported features in the library.

How can I tell?  In the debug logs I see:

`
2023-04-13 09:37:58.658 DEBUG (MainThread) [arcam.fmj.client] Packet received: AmxDuetResponse(values={'Device-SDKClass': 'Receiver', 'Device-Make': 'JBL SYNTHESIS', 'Device-Model': 'SDP-58', 'Device-Revision': '2.0.0'})
`

**Comment by elupus** [comment]
Right that looks like what i was looking for. If you see here: https://github.com/elupus/arcam_fmj/blob/master/src/arcam/fmj/state.py#L379 we map these responses to different arcam series of receivers. It affect things like available sources and how to activate sources.

If you have the rs232 docs for that receiver you can likely figure out which it matches most closely.

**Comment by kernelpanic85** [comment]
> Right that looks like what i was looking for. If you see here: https://github.com/elupus/arcam_fmj/blob/master/src/arcam/fmj/state.py#L379 we map these responses to different arcam series of receivers. It affect things like available sources and how to activate sources.
> 
> If you have the rs232 docs for that receiver you can likely figure out which it matches most closely.

Thanks for the tip!  It looks like the JBL SDP-55 and SDP-58 fall into the APIVERSION_HDA_SERIES.  And actually it's not mapping correctly right now for me even though basic functionality works.  

Is this just a matter of adding the models to this?  I can make a PR for it.
https://github.com/elupus/arcam_fmj/blob/master/src/arcam/fmj/__init__.py

**Comment by elupus** [comment]
Should be yes.

**Pull request merged**